js/main.js

```diff
@@ -0,0 +1,172 @@
+const $images = document.querySelector('.images');
+const $search = document.querySelector('.search');
+const $genreButton = document.querySelector('.genre');
+const $artistButton = document.querySelector('.artist');
+
+function formatDate(inputDate) {
+  const parts = inputDate.split('-');
+  const year = parts[0];
+  const month = parts[1];
+  const day = parts[2];
+  return `${month}/${day}/${year}`;
+}
+
+function showSearchBar() {
+  $search.classList.remove('hidden');
+}
+
+function getImage(DMA) {
+  const xhr = new XMLHttpRequest();
+  xhr.open('GET', `https://app.ticketmaster.com/discovery/v2/events.json?classificationName=music&dmaId=${DMA}&apikey=aeMvG0zyzdpO1jAkGyCZeGxxQK4vIfpe`);
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    $search.classList.add('hidden');
+    const response = xhr.response._embedded.events;
+    for (let i = 0; i < response.length; i++) {
+      const $imgWrapper = document.createElement('div');
+      const $img = document.createElement('img');
+
+      const $artistName = document.createElement('p');
+      $artistName.textContent = response[i].name;
+
+      const $venue = document.createElement('p');
+      $venue.textContent = response[i]._embedded.venues[0].name;
+
+      const $date = document.createElement('p');
+      const formattedDate = formatDate(response[i].dates.start.localDate);
+      $date.textContent = `${formattedDate}`;
+
+      for (let j = 0; j < response[i].images.length; j++) {
+        const currentImage = response[i].images[j];
+        if (currentImage.ratio === '4_3') {
+          $img.src = currentImage.url;
+        }
+      }
+
+      $imgWrapper.appendChild($img);
+      $imgWrapper.appendChild($artistName);
+      $imgWrapper.appendChild($venue);
+      $imgWrapper.appendChild($date);
+      $imgWrapper.classList.add('column-third');
+      $images.appendChild($imgWrapper);
+
+    }
```

Comment:
You seem to repeat this code multiple times, so I would break this up into it's own function. Make sure the actual appending to `$images` happens where you call the function though